source/nextPlayer.py

```diff
@@ -0,0 +1,11 @@
+def nextPlayer(players, current_player, drawCard):
+    #creates an array out of players. 
+    num_players = len(players)
+    if drawCard:
+        #if a card is drawn the turn goes to the next player in the array. 
+        next_player = (current_player + 1) % num_players
+        return next_player
+    else:
+        #if a card isn't drawn the player goes again. 
+        return current_player
+
```

Comment:
Great code, one thing is that there could be a problem when it comes to getting a card from another player, It doesn't look like the code would count that as a turn so it might just stay as the same players turn throwing off the number of cards in the deck or he might have 1 too many.

Reply:
Good point. Not sure if we have a function for that already. If not we can integrate that into this function or create a separate one for it.

Reply:
The function looks good Andre, I like how you used an array for the player position.

Reply:
> The function looks good Andre, I like how you used an array for the player position.

Thanks for the review, Zach! I'm glad you were able to see and understand my approach.